spec/integrations/base_spec.rb

```diff
@@ -234,6 +234,31 @@ def extract(association_name, object, _local_options, _options={})
           end
         end
       end
+
+      context "Given association is empty" do
+        before do
+          empty_relation = obj.vehicles
+          expect(obj).to receive(:vehicles).and_return(empty_relation)
```

Comment:
I think this assertion is unnecessary. We should know the state of our testing environment at this point.

If this expectation failed, the later expectations would also fail, correct?

Reply:
You're right, I have removed the useless assertion!